keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/Environment.kt

```diff
@@ -4,6 +4,7 @@ data class Environment(
   val name: String,
   val resources: Set<Resource<*>> = emptySet(),
   val constraints: Set<Constraint> = emptySet(),
+  val verifyWith: Set<Validation> = emptySet(),
```

Comment:
@lorin we discussed the order of evaluation of things in here. I've made it a set for now given we use that almost everywhere else but do you think it would make sense to rather use a list?

Reply:
I'm inclined to go with a list, given that the majority of the people who we've asked assume that the validations would run in the order defined in the delivery config.

Reply:
@robfletcher Why did you choose "verifyWith" instead of "validateWith"?

Reply:
Yeah,  I wondered because I started thinking about how we'll record verification state in the database.